tests/vmcheck/test-layering-scripts.sh

```diff
@@ -154,8 +154,18 @@ fi
 
 # Should work now that we're using --copyup
 # https://github.com/projectatomic/rpm-ostree/pull/1171
+test_overwritepath=
+for path in /usr/share/licenses/glibc/COPYING
+            /usr/share/systemd/kbd-model-map
+            /usr/share/rpm-ostree/treefile.json; do
```

Comment:
Hmm, these lines need to have their newlines escaped, no?
Let's just use `/usr/share/rpm-ostree/treefile.json` for all cases?